### PR TITLE
context-delete Unit Tests

### PR DESCRIPTION
**Pull request by sanatnayar**
Unit tests for: `qliksense config delete-context <context>`

### CODE REVIEW

File: pkg/qliksense/context_configs.go [hunk 1]
```diff
@@ -287,8 +287,12 @@ func (q *Qliksense) DeleteContextConfig(args []string) error {
 		case qliksenseConfig.Spec.CurrentContext:
 			fmt.Fprintln(out, chalk.Red.Color("Error: Cannot delete current context -"), chalk.Bold.TextStyle(qliksenseConfig.Spec.CurrentContext))
 			fmt.Fprintln(out, chalk.Yellow.Color("Please switch contexts to be able to delete this context."))
+			err := fmt.Errorf("")
```
Comment (andriy-bulynko) [inline pkg/qliksense/context_configs.go]: This is a bit confusing to me.. So, we're telling upstream that we have an error here, but not telling them what it is..

File: pkg/qliksense/context_configs.go [hunk 2]
```diff
@@ -287,8 +287,12 @@ func (q *Qliksense) DeleteContextConfig(args []string) error {
...
+			return err
 		case DefaultQliksenseContext:
 			fmt.Fprintln(out, chalk.Red.Color("Error: Cannot delete default qliksense context"))
+			err := fmt.Errorf("")
```
Comment (andriy-bulynko) [inline pkg/qliksense/context_configs.go]: :)

File: pkg/qliksense/context_configs.go [hunk 3]
```diff
@@ -322,6 +326,9 @@ func (q *Qliksense) DeleteContextConfig(args []string) error {
 						fmt.Fprintln(out, chalk.Green.Color("Successfully deleted context: "), chalk.Bold.TextStyle(args[0]))
 					} else {
 						fmt.Fprintln(out, chalk.Red.Color("Error: Context not found"))
+						err := fmt.Errorf("")
```
Comment (andriy-bulynko) [inline pkg/qliksense/context_configs.go]: :)

**Review by andriy-bulynko — APPROVED** [review]

File: pkg/qliksense/context_configs.go (hunk 1 above)
Comment (sanatnayar) [reply pkg/qliksense/context_configs.go]: I added the error invocation specifically so the test cases can catch any error. I will try and put the actual error in that as well but I had specific formatting (color coding) that I dont think Cobra's error handling will be able to print out correctly.

**Review by ashwathishiva — APPROVED** [review]

File: pkg/qliksense/context_configs.go (hunk 1 above)
Comment (sanatnayar) [reply pkg/qliksense/context_configs.go]: fixed it :) should format correctly

File: pkg/qliksense/context_configs.go (hunk 2 above)
Comment (sanatnayar) [reply pkg/qliksense/context_configs.go]: fixed it :) should format correctly

File: pkg/qliksense/context_configs.go (hunk 3 above)
Comment (sanatnayar) [reply pkg/qliksense/context_configs.go]: fixed it :) should format correctly

**Pull request merged**